app/main.py

```diff
@@ -1 +1,51 @@
-#  write your code here
+class Animal:
+
+    def __init__(self,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one,